app/src/main/java/com/hamdan/forzenbook/view/login/CreateAccount.kt

```diff
@@ -368,3 +306,15 @@ private fun Content(
         }
     }
 }
+
+fun stringDate(month: Int, day: Int, year: Int, context: Context): String {
+    return if (month < 10) {
+        if (day < 10) {
+            context.getString(R.string.create_account_date, month, day, year)
+        } else context.getString(R.string.create_account_date_day_tens, month, day, year)
+    } else {
+        if (day < 10) {
+            context.getString(R.string.create_account_date_month_tens, month, day, year)
+        } else context.getString(R.string.create_account_date_month_day_tens, month, day, year)
```

Comment:
![image](https://user-images.githubusercontent.com/112203610/216547319-36ff9c9b-acdb-43a4-bb29-0a0be8efd007.png)

Reply:
You can make an extension function for left padding